docs/configuration-customizations.md

```diff
@@ -58,6 +58,38 @@ customization:
     background-base: '#fbfbfb;'
 ```
 
+## Visualisation colors
+
+Turnilo allows you to define colors for charts.
+
+For example:
+
+```yaml
+customizaiton:
+  visualizationColors: 
+    main: #FF5900
+    series:
+      - #2D95CA
+      - #EFB925
+      - #DA4E99
+      - #4CC873
+      - #745CBD
+      - #EA7136
+      - #E68EE0
+      - #218C35
+      - #B0B510
+      - #904064
+```
+
+Value is any CSS Color Module Level 3 specifier string. 
+Object is merged with default values, so it is possible to override just one key.
```

Comment:
1. Could you refer to the source with default values (or specify default colors here), please? 
2. I don't understand how built-in colors are merged with the colors in configuration file, especially series array.
3. I would add a note to CSS variables section about visualization colors. For example: cssVariables.brand should be changed together with visualizationColors.main to get consistent look.